File: backend/CargoTrack.Api/Entities/City.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CargoTrack.Api.Entities;

[Table("city")]
public class City
{
    [Key]
    [Column("cityid")]
    public int CityId { get; set; }
        
    [Column("name")]
    public required string Name { get; set; }
}

File: backend/cargoTrack/src/main/java/backend/cargoTrack/model/City.java
package backend.cargoTrack.model;

import jakarta.persistence.*;
@Entity
@Table(name = "city")
public class City {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_city")
    private int idCity;
    @Column(name = "name")
    private String name;
    @Column(name = "population")
    private int population;
    @Column(name = "coordinates")
    private String coordinates;

}
